### ISSUE TITLE
Add model comparison (RandomForest, XGBoost, LightGBM)

### ISSUE THREAD

**Issue by Pratikchetry** [Pratikchetry/Customer_Churn_LTV_Engine#1]
Compare multiple ML models and select the best one based on accuracy.
